src/components/Background.jsx

```diff
@@ -0,0 +1,18 @@
+import FAQ from './FAQ.jsx';
+import './Background.css'
+
+const Background = () => {
+    return (
+        <div className="background">
+            <div className="illustration_container">
+                <div className="main">
+                    <FAQ />
+                    <img src="/images/bg-pattern-desktop.svg" className="bg-img" alt="" />
+                    <img src="/images/illustration-woman-online-desktop.svg" className="illustration" alt="illustration" />
```

Comment:
`alt` text should be something like `woman-online`. Describe contents, not what it is